src/app/clusters/application-basic-server/application-basic-delegate.h

```diff
@@ -42,6 +42,7 @@ class Delegate
     virtual ApplicationStatusEnum HandleGetStatus()                                                  = 0;
     virtual chip::CharSpan HandleGetApplicationVersion()                                             = 0;
     virtual std::list<uint16_t> HandleGetAllowedVendorList()                                         = 0;
+    virtual const char * GetApplicationName()                                                        = 0;
```

Comment:
This is probably duplicate as there's already this method:
`virtual chip::CharSpan HandleGetApplicationName()`

Reply:
removed method